packages/access-client/src/agent.js

```diff
@@ -54,25 +55,29 @@ async function createIssuerSaysAccountCanAdminSpace(
       with: space,
     },
   ],
-  proofs = []
+  proofs = [],
+  expiration = Infinity
```

Comment:
can we either:
* not default this to Infinity (it's a risky deafult)
* since this function is kinda just a wrapper around `ucanto.delegate` and iirc we only use this function one time, can we just inline this all at point of use?

Reply:
I think keeping them as separate functions is a huge win for readability so I'll just get rid of this default